modules/yosys/0.57/presubmit.yml

```diff
@@ -0,0 +1,16 @@
+bcr_test_module:
```

Comment:
With the test module, it's actually the CI script doing the archive extracting and patching, see at https://github.com/bazelbuild/continuous-integration/blob/44b73146f92c9c03b45427348f54ee8fc4c0a22b/buildkite/bazel-central-registry/bcr_presubmit.py#L224

Looks like the `patch` tool on macOS didn't successfully remove the BUILD files.

Reply:
Sounds like a bug to me. Especially since it works just fine on linux. But I've gone with a different approach.

Reply:
Yeah, this inconsistence is causing trouble from time to time, I'm planning to use `bazel vendor` to extract the sources instead.